apps/pythinker-web/src/components/settings/settings.css

```
@@ -61,10 +61,11 @@
 .act.signin { background: var(--blue); color: var(--bg); border-color: var(--blue); }
 .act.signin:hover { background: var(--blue2); }
 .switch {
+  --switch-knob-size: calc(var(--ui-font-size) + 4px);
   position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)